sass/routes/_admin-console.scss

```diff
@@ -581,10 +581,12 @@
             }
         }
         .input-clear {
-            color: $white;
-            font-weight: bold;
-            position: absolute;
-            right: 16px
+            .input-clear-x {
+                color: $white;
+                font-weight: bold;
+                position: absolute;
+                right: 16px
+            }
```

Comment:
I did want to make a quick comment that this is not needed if you wanted to leave it out. Scoping the first CSS like you've done will fix it and you don't need to make this change hence fixing the regression that Scott pointed out without touching this file. I'll leave it up to you as either way works for me.

Reply:
I'm ok with leaving this in. It will protect it if the same issue happens again in the future.